oasislmf/manager.py

```diff
@@ -422,7 +423,7 @@ def generate_oasis_files(
         )
 
         # If not in det. loss gen. scenario, write exposure summary file
-        if not deterministic:
+        if not deterministic and summarise_exposure:
```

Comment:
@hchagani-oasislmf Looks fine, but can you please make that change to the help text (https://github.com/OasisLMF/OasisLMF/pull/323/files/dd5f3555aae65f3da120b9cbb1deb3f9da1402ce#r298207663), and also rewrite this conditional as follows:

    if summarise_exposure and not deterministic:

I think the conditional is a bit ambiguous if it started with checking for `deterministic`.

Reply:
@sr-murthy Done on both counts. Pushed new files to repo.